### ISSUE TITLE
`kms` doesn't reject ARNs without `awskms://`

### ISSUE THREAD

**Issue by mattmoor** [sigstore/policy-controller#523]
**Description**

Debugging a customer issue we noticed that:
```yaml
   kms: arn:...
```

... is not rejected by the validating webhook, and that the error message with `awskms://` vs. `awskms:///` (note: triple `/`!) is VERY unclear, basically what this unit test checks around "too many colons":
https://github.com/sigstore/policy-controller/blob/e386c060086fd9d68af0965258d88b8a5b69a972/pkg/apis/policy/v1alpha1/clusterimagepolicy_validation_test.go#L1307

IMO both of these should have very clear error messages that:
1. Indicate a scheme is required, and
2. Indicate that a `/` is missing instead of complaining about colons.

**Version**

0.5.2

**Comment by hectorj2f** [comment]
Yes, it makes sense. Thanks for opening the issue.

**Comment by hectorj2f** [comment]
I proposed some changes in https://github.com/sigstore/policy-controller/pull/524. 

On the other hand, after looking at the awskms validations, I realized the test case  `errorString: "invalid value: awskms://localhost:::4566/alias/exampleAlias: KMSORCACERT\nmalformed endpoint: address localhost:::4566: too many colons in address",` sounds like a valid error message. The validation assumes valid format are provided and  `awskms://localhost:::4566/alias/exampleAlias:` is a valid format from a general regex perspective. However when the func validates the endpoint part from the `awskms://[ENDPOINT]/[ID/ALIAS/ARN] (endpoint optional)` format, it fails with a valid error message in my opinion.
